README.md

```diff
@@ -1,25 +1,38 @@
-# Forks and Clones
+# Forking and Cloning Github Repositories
+
+## Problem Statement
+
+It will be important to get used to forking and cloning repos. Both are `git` functions
+that allow a user to make copies of the repo, but can serve different functions, depending
+on what needs to be done. If you're working on a repository that you don't have "write"
+access to, such as Flatiron School's `github.com/learn-co-students/` repo, you may find
+yourself cloning a readme or lab to your local environment, but being unable to submit
+your changes with a `git push`. How are you able to keep track of your changes that are
+being added to a repo you're working on? This is where you would use `git fork` and make
+a clone of that instead!
 
 ## Objectives
 
-1. Fork a repo
-2. Clone a repo
+1. Use `git fork` to copy a repo
+2. Use `git clone` to save a repo to your local machine
 
-## Forking a GitHub Repository
+## Use `git fork` to Copy a Repo
```

Comment:
( Review notes here assume you've re-ordered in line with the objectives I listed above )